databrowser/src/domain/datasets/ui/toolBox/export/ExportDatasetsToolBoxPanel.vue

```diff
@@ -145,12 +156,16 @@ const onCopyReference = (url: string) => {
   copyReferenceUrl(url);
 };
 
-const { copy: copyReferenceUrl, copied: copiedReferenceUrl } = useClipboard();
+const { copy: copyReferenceUrl } = useClipboard();
```

Comment:
This mechanism doesn't reset the icon from `IconCheck` to `IconCopy` after some time, like it is the case with `copyUrl` / `copiedUrl`. Note, that this bug exists also in the old code, it would be nice if we could fix it in this iteration